src/DynamoCore/Interfaces/IPackage.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Dynamo.Interfaces
+{
+    /// <summary>
+    /// Interface that contains basic properties of a Dynamo package
+    /// </summary>
+    public interface IPackage
```

Comment:
this must exist somewhere else? Is it a case where you don't want to add a reference between projects?

Reply:
Yeah, we can't reference Package Manager in Core. I can't find it anywhere else.